### PR TITLE
This Pr got closed so I'm gonna have fun with the title :honkman: :revolver: :silentman:

### PR DESCRIPTION
**Pull request by ghost**
**Do you like hurting other people?**
If the answer to this question is yes, put a thumbs-up emoji in the comments below.

This PR adds three things:
1. Weapons inspired by Hotline Miami; only one is obtainable normally for now, the others are admin-spawn
2. A voice synthesizer which speaks what you enter into it; basically lets you talk while disguised. Only obtainable through traitor bundle at the moment
3. A new traitor bundle: the Jacket bundle. Included is a voice synthesizer, chicken mask, bomber jacket, classic jeans, frying pan, revolver, and Syndicate first-aid kit (that last is subject to change).

Weapon sprites are from WalterJe's sprite thread, with two-handed sprites made by me. The voice synthesizer tape sprite was made by me; it is a codersprite and anyone who wants to do better is encouraged to, since this is what it looks like currently.

recorder1

For the record, I wanted to make custom masks but I'm terrible at spriting and I didn't have the patience for trying to make them look at least semi-decent. If sprites are provided, I _will_ add animal masks.

Weapon descriptions allude to the names of songs in the soundtrack of Hotline Miami (Hydrogen, Paris, Inner Animal, etc.)

I can add more screenshots if it is requested.

### CODE REVIEW

**Comment by hornygranny** [comment]
👮📧➡

**Comment by dannno** [comment]
lol how though hg
i mean yeah he could have been more subtle about it but

**Comment by Xhuis** [comment]
There's nothing that directly alludes to the game itself, only references.

**Comment by hornygranny** [comment]
look at your commit messages and paths

**Comment by Xhuis** [comment]
I can fix that if it bothers you so much.

**Comment by dannno** [comment]
make a new one i guess then?

File: code/game/objects/items/weapons/miami.dm [hunk 1]
```diff
@@ -0,0 +1,111 @@
+/obj/item/weapon/miami
+	name = "base hotline miami weapon"
```
Comment (dannno) [inline code/game/objects/items/weapons/miami.dm]: ok yeah this was pretty stupid

File: code/game/objects/items/weapons/miami.dm [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	w_class = 1
+	throw_speed = 2
+	throw_range = 7
+	force = 10
+	m_amt = 10
+	g_amt = 10
+
+/obj/item/weapon/miami/knife
+	name = "pocket knife"
+	desc = "A small stainless-steel pocket knife. It's as sharp as the edge of a flatline."
```
Comment (dannno) [inline code/game/objects/items/weapons/miami.dm]: this is a sick desc tho

Comment (Xhuis) [inline code/game/objects/items/weapons/miami.dm]: References the track "Flatline" in the soundtrack

**Comment by Xhuis** [comment]
Stuff is fixed but not going onto the branch for some reason, even after push
Should I just make another PR?

**Comment by hornygranny** [comment]
make a new pr on a new branch

**Comment by Xhuis** [comment]
k

**Pull request closed without merging**